message.go

```diff
@@ -456,7 +456,7 @@ func (r RedisResult) ToMap() (v map[string]RedisMessage, err error) {
 	if r.err != nil {
 		err = r.err
 	} else {
-		v, err = r.val.ToMap()
+		v, err = r.val.AsMap()
```

Comment:
Why do we need this change? I think we already have an `AsMap` delegation.

Reply:
This is accidentally committed, I've reverted it.